src/styles.scss

```diff
@@ -28,7 +28,8 @@ body {
   margin-bottom: 0.5rem;
 }
 
-app-sign-up {
+app-sign-up,
+app-sign-in {
```

Comment:
slippery way )) no one will find these rules here

Reply:
ok. so what are you suggesting? ng-deep is deprecated. :host?

Reply:
ng-deep is being deprecated for 7 versions already ... i guess no one will remove it any soon. And my guess you don't need ng-deep when you will fix comment with setting custom material styles

Reply:
fixed